GPU/GPUTracking/Definitions/GPUDefGPUParameters.h

```diff
@@ -81,6 +81,7 @@
   #define GPUCA_LB_GPUTPCCFNoiseSuppression 512
   #define GPUCA_LB_GPUTPCCFDeconvolution 512
   #define GPUCA_LB_GPUTPCCFClusterizer 448
+  #define GPUCA_LB_GPUTPCNNClusterizerKernels 448
```

Comment:
Since you don't found tuned values for all the architectures yet, it would be better to just leave the default below
```
  #ifndef GPUCA_LB_GPUTPCNNClusterizerKernels
    #define GPUCA_LB_GPUTPCNNClusterizerKernels 512
  #endif
```
and remove the 3 architecture-dependent lines here.